.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+
+install: "pip install flask"
+
+services:
+  - docker
```

Comment:
Please add any `script`-key to make travis pass. (`true` should be enough as a command)